components/dashboard/src/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import Modal from "./Modal";
+
+export default function ConfirmationModal(props: {
+    title?: string;
+    areYouSureText?: string,
```

Comment:
**praise**: Nice one! Thanks for adding this abstraction based on the existing modals. 🔝